### PR TITLE
V2.4 Door Hinge M3 as shaft,  3mm panel, 180deg opening freedom

### PR DESCRIPTION
**Pull request by quattroerik**

### CODE REVIEW

**Comment by cbc02009** [comment]
Can you add a bit more instruction about the install process? where the heatset inserts go, what to use to attach to frame, etc?

**Comment by quattroerik** [comment]
Sure, i have added some additional pictures about the assembly and how to attach to the frame. (no changes to the original ones..)

**Review by cbc02009 — APPROVED** [review]

File: printer_mods/README.md [hunk 1]
```diff
@@ -225,6 +225,7 @@ like so:
 |piet206 |[V1.8 Z Rails Mod](./piet206/V1.8._Z_Rails_Mod) | Add MGN12H linear rails for Z axis motion |:x: |:heavy_check_mark: |:x: |:x:|
 | Printopal | [Magnetic top panel](Printopal/Magnetic_top_panel) | Top panel with magnets allows for quick installation and removal | :x: | :heavy_check_mark:| :heavy_check_mark: | :x: |
 |QuackProfit|[V0 Adafruit NeoPixel LED Mounts](./QuackProfit/V0-Adafruit-Y_Rails) | Mount Adafruit PCB NeoPixel Sticks on the V0 Y-rail extrusions | :heavy_check_mark:|:x:|:x:|:x:|
+| quattroerik | [V2.4 Door Hinge M3 3mm 180deg](./quattroerik/front_door_m3_180_deg_3mm) | Voron 2.4 door hinges for 3mm panel, 180 deg opening and m3 screw as shaft | :x: | :x: | :heavy_check_mark: | :x: |
```
Comment (cbc02009) [inline printer_mods/README.md:228]: Can you add a ✔️ for the trident column?

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by quattroerik** [comment]
Added checkmark for trident

File: printer_mods/README.md [hunk 2]
```diff
@@ -246,7 +246,7 @@ like so:
 |piet206 |[V1.8 Z Rails Mod](./piet206/V1.8._Z_Rails_Mod) | Add MGN12H linear rails for Z axis motion |:x: |:heavy_check_mark: |:x: |:x:|
 | Printopal | [Magnetic top panel](Printopal/Magnetic_top_panel) | Top panel with magnets allows for quick installation and removal | :x: | :heavy_check_mark:| :heavy_check_mark: | :x: |
 |QuackProfit|[V0 Adafruit NeoPixel LED Mounts](./QuackProfit/V0-Adafruit-Y_Rails) | Mount Adafruit PCB NeoPixel Sticks on the V0 Y-rail extrusions | :heavy_check_mark:|:x:|:x:|:x:|
-| quattroerik | [V2.4 Door Hinge M3 3mm 180deg](./quattroerik/front_door_m3_180_deg_3mm) | Voron 2.4 door hinges for 3mm panel, 180 deg opening and m3 screw as shaft | :x: | :x: | :heavy_check_mark: | :x: |
+| quattroerik | [V2.4 Door Hinge M3 3mm 180deg](./quattroerik/front_door_m3_180_deg_3mm) | Voron 2.4 door hinges for 3mm panel, 180 deg opening and m3 screw as shaft | :x: | :x: | :heavy_check_mark: | :heavy_check_mark: |
```
Comment (cbc02009) [inline printer_mods/README.md:249]: Please dd a 5th column for trident. you updated the switchwire value.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/README.md (hunk 2 above)
Comment (quattroerik) [reply printer_mods/README.md]: Sorry! -> done :-)

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**